Makefile

```diff
@@ -0,0 +1,49 @@
+pip:
+	pip install -e .
```

Comment:
Why would this ever be used over `pip-development`? If you're installing the package locally, you likely want the development dependencies as well. And `pip install -e .[development]` also install non-development dependencies, so there's no advantage to doing this first.

Reply:
👍